code/_globalvars/traits/_traits.dm

```diff
@@ -153,6 +153,7 @@ GLOBAL_LIST_INIT(traits_by_type, list(
 		"TRAIT_CLUMSY" = TRAIT_CLUMSY,
 		"TRAIT_COAGULATING" = TRAIT_COAGULATING,
 		"TRAIT_CORPSELOCKED" = TRAIT_CORPSELOCKED,
+		"TRAIT_CRIMINAL_SYMPATHISER" = TRAIT_CRIMINAL_SYMPATHISER,
```

Comment:
abc

Reply:
what am i missing isnt this already in the correct alphabetical location?

Reply:
my eyes deceived me